metabase/metabase
# sunburst improvements

Thread https://metaboat.slack.com/archives/C064QMXEV9N/p1727976022922249 

### Description

Making two improvements to the sunburst chart

1. Percentages on the chart and in the tooltip should be relative to the parent slices' total, not the overall total. These changes can be found in `echarts/pie/model/index.ts` and `visualizations/PieChart/use-chart-events.ts`

2. Dimension picker hover styles should match the pie slices widget style. Particularly, we should show the grab cursor when hovering over the dimension picker, and also apply brand color to the handle icon.

### How to verify

1. Create a sunburst chart
2. Enable percentages on chart, confirm values match expectations on chart and tooltip
3. Hover over the dimension picker for each ring, confirm styles are as expected

### Demo

![image](https://github.com/user-attachments/assets/72f60842-8e40-4996-bb12-bb8e937194ac)


https://github.com/user-attachments/assets/73b67f33-46fe-439b-9a53-8b10d072763c


https://github.com/user-attachments/assets/ea0997b5-77a1-47fa-b307-73ed925064ff

Confirming the component styles are correct on line chart as well

### Checklist
